persistence/src/commonMain/db_user/com/wire/kalium/persistence/Users.sq

```diff
@@ -299,3 +299,10 @@ UPDATE User SET team = ? WHERE qualified_id = ?;
 selectNameByMessageId:
 SELECT name FROM User
 WHERE qualified_id = (SELECT Message.sender_user_id FROM Message WHERE Message.id = :messageId AND Message.conversation_id = :conversationId);
+
```

Comment:
we would need to add index to the connection status in this case to support a wider range of use cases we can use 
```
CREATE INDEX idx_user_connection_deleted_handle ON User (connection_status, deleted, handle);
```

this will make it also faster to do search 2 birds in 1 stone